classifiers/image/CLIPZeroShotClassifier/__init__.py

```diff
@@ -0,0 +1,57 @@
+from typing import List
+
+import torch
+from jina.executors.classifiers import BaseClassifier
+from jina.executors.decorators import as_ndarray, batching
+from jina.executors.devices import TorchDevice
+from PIL import Image
+
+
+class CLIPZeroShotClassifier(TorchDevice, BaseClassifier):
+    """
+    :class:`ClipZeroShotClassifier` Zero Shot classification for images using OpenAI Clip.
+
+    Internally, :class:`ClipZeroShotClassifier` wraps the `CLIP` modeL from https://github.com/openai/CLIP
+    :param labels: labels for the classification task. 
+    :param model_name: The name of the model. Supported models include ``ViT-B/32`` and ``RN50``.
+    :param args: Additional positional arguments.
+    :param kwargs: Additional positional arguments.
+    """
+
+    def __init__(self, labels: List[str], model_name: str ='ViT-B/32',
```

Comment:
Hey @deepampatel ,

Could you please provide a valid default value for labels. We need this for proper testing.

I think that due to some bug this PR is not being properly tested

Reply:
@JoanFM  i will add default labels. Seems the ci hub build is not called because `classifiers` folder is not tracked https://github.com/jina-ai/jina-hub/blob/master/.github/workflows/ci.yml#L29

Reply:
good finding, we need to add this definitely

Reply:
i was trying with CI and adding default labels on a fork pr. https://github.com/deepampatel/jina-hub/pull/1
seems the   flow stalls as soon as the request is received can you help @JoanFM

Reply:
Can u try running this from the `folder` of this executor?

`jina hub build --prune-images --raise-error --host-info --test-uses --daemon --timeout-ready 80000 .`

What does it happen?

Reply:
For me, locally it failed.
```bash
Traceback (most recent call last):
  File "/Users/nanwang/Codes/jina-ai/jina/jina/peapods/runtimes/zmq/zed.py", line 73, in _load_executor
    self.args.uses, pea_id=self.args.pea_id, read_only=self.args.read_only
  File "/Users/nanwang/Codes/jina-ai/jina/jina/jaml/__init__.py", line 499, in load_config
    return JAML.load(tag_yml, substitute=False)
  File "/Users/nanwang/Codes/jina-ai/jina/jina/jaml/__init__.py", line 89, in load
    r = yaml.load(stream, Loader=JinaLoader)
  File "/Users/nanwang/.pyenv/versions/3.7.5/lib/python3.7/site-packages/yaml/__init__.py", line 114, in load
    return loader.get_single_data()
  File "/Users/nanwang/.pyenv/versions/3.7.5/lib/python3.7/site-packages/yaml/constructor.py", line 51, in get_single_data
    return self.construct_document(node)
  File "/Users/nanwang/.pyenv/versions/3.7.5/lib/python3.7/site-packages/yaml/constructor.py", line 55, in construct_document
    data = self.construct_object(node)
  File "/Users/nanwang/.pyenv/versions/3.7.5/lib/python3.7/site-packages/yaml/constructor.py", line 100, in construct_object
    data = constructor(self, node)
  File "/Users/nanwang/Codes/jina-ai/jina/jina/jaml/__init__.py", line 399, in _from_yaml
    return get_parser(cls, version=data.get('version', None)).parse(cls, data)
  File "/Users/nanwang/Codes/jina-ai/jina/jina/jaml/parsers/executor/legacy.py", line 131, in parse
    requests=data.get('requests', {}),
  File "/Users/nanwang/Codes/jina-ai/jina/jina/executors/__init__.py", line 54, in __call__
    obj = type.__call__(cls, *args, **kwargs)
TypeError: __init__() missing 1 required positional argument: 'labels'
```